selection.Rmd

```diff
@@ -239,9 +239,9 @@ lines(xplot, predict(fit_big_removed, newdata = data.frame(x = xplot)),
       col = "darkorange", lwd = 2, lty = 2)
 ```
 
-We see that on average, the solid blue line for the quadratic model has similar errors as before. It has changed very slightly. However, the dashed orange line for the large model, has a huge error at the point that was removed and is much different that the previous fit.
+We see that on average, the solid blue line for the quadratic model has similar errors as before. It has changed very slightly. However, the dashed orange line for the large model, has a huge error at the point that was removed and is much different from the previous fit.
```

Comment:
If it is OK grammatically, I think I prefer "than" which I think is what I tried for, but made a typo. I'm OK with "from" though.

Reply:
I updated the content, so please check this commit: https://github.com/daviddalpiaz/appliedstats/pull/128/commits/abfbf1800fe76f411501f530b10df2d6473c319f